tests/performance/test_mongo_redis_performance.py

```diff
@@ -0,0 +1,197 @@
+import random
```

Comment:
Although we'll not run a more restrict lint tool in tests by now, I believe you should use proper type hinting whenever possible. You did it in some methods but didn't in others.

Reply:
Modified to support the lint tools